recipe/meta.yaml

```
@@ -25,28 +28,29 @@ requirements:
   host:
     # Host dependencies are pinned to last build
     - python
-    - cython 3.0.8
+    - cython >=3.0.10
     - numpy {{ numpy }}
     - llvm-openmp 14.0.6  # [osx]
     - pip
-    - scipy >=1.6.0
-    - setuptools
-    - wheel
+    - scipy  1.9.3        # [py<312]
+    - scipy  1.11.3       # [py>=312]
+    - meson-python >=0.15.0
```

```suggestion
    - meson-python >=0.16.0
```
https://github.com/scikit-learn/scikit-learn/blob/1.5.1/pyproject.toml#L100

---

That's fair, my tooling was stuck on 1.5.0

---

Duh!  Apart from the fact we don't have `meson-python 0.16.0`, only `0.15.0`.

There doesn't appear to be any substantive difference but you may have a better eye for these things.

---

`meson-python` `1.16.0` even fixes the reliability issues for `win-64` `py312`